### ISSUE TITLE
Rule request: replacing unwanted comment substrings with valid ones

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [microsoft/tslint-microsoft-contrib#464]
Suppose your company _hates_ when people call it "XBox", and wants to replace that with "Xbox":

```typescript
/**
 * Something something XBox something.
 */
export const value = 7;
```

Perhaps it would be useful to have a TSLint rule to flag barred comment values, and suggest replacing them?

```typescript
/**
 * Something something XBox something.
                    // ^^^^ Replace with: "Xbox"
 */
export const value = 7;
```

Some other useful things:
* Replacing " his " with " their " (for fixing common gender neutrality typos)
* Replacing words commonly flagged by tools such as [PoliCheck](http://archives.miloush.net/michkap/archive/2010/12/04/10100294.html)

Edit: prototyped this internally; using `tsutils.forEachComment`, it's relatively straightforward.

**Comment by JoshuaKGoldberg** [comment]
### 💀 _**It's time!**_ 💀

**TSLint is being deprecated**; therefore, it and tslint-microsoft-contrib are no longer accepting pull requests for major new changes or features. See #4534. 😱

If you'd like to see this change implemented, you have two choices:
* **Recommended:** Check if this is available in [ESLint](https://eslint.org) + [typescript-eslint](https://typescript-eslint.io) ✅
* _Not Recommended: Fork TSLint or tslint-microsoft-contrib locally_ 🤷‍♂️

👋 It was a pleasure open sourcing with you!

_If you believe this message was posted here in error, please comment so we can re-open the issue!_
